pkg/asset/tls/kubelet.go

```diff
@@ -48,6 +48,7 @@ var _ asset.Asset = (*KubeletClientCABundle)(nil)
 func (a *KubeletClientCABundle) Dependencies() []asset.Asset {
 	return []asset.Asset{
 		&KubeletCSRSignerCertKey{},
+		&KubeAPIServerToKubeletCABundle{},
```

Comment:
This looks correct.  This is the bundle that can be used to verify the kube-apiserver's client cert to the kubelet.